app/views/layouts/angular-bootstrap/_endpoints_angular.html.haml

```diff
@@ -78,6 +79,17 @@
                           "checkchange"             => "",
                           "prefix"                  => prefix.to_s,
                           "reset-validation-status" => "#{prefix}_auth_status"}
+    - detect = "detectClicked({target: '.detect_button'})"
+    %span
+      %miq-button.detect_button{"ng-if" => detection ? 'true' : 'false',
+                  :name            => _("Detect"),
+                  "on-click"       => detect,
+                  :enabled         => "isDetectionEnabled()",
+                  "disabled-title" => _("Detection not available for provider type"),
```

Comment:
is this the reason it would be visible but disabled?
I think for other providers you just hide it & this should explain this fields you have to fill.